src/courtroom.cpp

```diff
@@ -3510,6 +3513,7 @@ void Courtroom::on_ooc_toggle_clicked()
 // Todo: multithread this due to some servers having large as hell music list
 void Courtroom::on_music_search_edited(QString p_text)
 {
+    ui_music_list->expandAll();
```

Comment:
Music list will be expanded every time you add anything to the search string regardless of your input as a user... Is this even good ui design???